src/main/java/com/mulberry/blackmulberry/BlackMulberryApplication.java

```diff
@@ -1,20 +1,46 @@
 package com.mulberry.blackmulberry;
 
+import com.mulberry.blackmulberry.entity.User;
+import com.mulberry.blackmulberry.entity.model.UserRole;
+import com.mulberry.blackmulberry.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
 
+import java.util.Optional;
+
 @SpringBootApplication
-public class BlackMulberryApplication {
+@RequiredArgsConstructor
+public class BlackMulberryApplication implements CommandLineRunner {
+    private final UserRepository userRepository;
+
+    public static void main(String[] args) {
+        SpringApplication.run(BlackMulberryApplication.class, args);
+    }
+
+    @Bean
+    public PasswordEncoder passwordEncoder() {
+        return new BCryptPasswordEncoder();
+    }
+
+    @Override
+    public void run(String... args) throws Exception {
 
+        Optional<User> byEmail = userRepository.findByEmail("admin@mail.com");
 
-	public static void main(String[] args) {
-		SpringApplication.run(BlackMulberryApplication.class, args);
-	}
-	@Bean
-	public PasswordEncoder passwordEncoder() {
-		return new BCryptPasswordEncoder();
-	}
+        if (byEmail.isEmpty()) {
+            userRepository.save(User.builder()
+                    .name("admin")
+                    .surname("admin")
+                    .email("admin@mail.com")
+                    .password(passwordEncoder().encode("admin"))
```

Comment:
i am not sure that its good idea to have admin credentials with open in code. so lets find another way. for excample you can get encrypted password and admin's data then add it in a sql insert with flyway. but i am sure that there are another ways too. 
if it spend more time , you can create separate ticket and will chenge later